.hound.yml

```diff
@@ -0,0 +1,5 @@
+javascript:
+  enabled: false
+eslint:
+  enabled: true
+  config_file: .eslintrc.json
```

Comment:
you are referencing a non-existing file `.eslintrc.json`

Reply:
I am waiting for your work to be merged and it will work!